api.planx.uk/modules/file/service/uploadFile.ts

```diff
@@ -1,6 +1,6 @@
 import S3 from "aws-sdk/clients/s3.js";
 import { customAlphabet } from "nanoid";
-import { getType } from "mime";
+import mime from "mime";
```

Comment:
We could probably keep the current functionality by upgrading to v4 which drops CJS https://github.com/broofa/mime/blob/v4.0.0/CHANGELOG.md

Reply:
Interestingly bumping `mime` doesn't actually enable us to revert this change (see issue #[295](https://github.com/broofa/mime/issues/295)), but I went ahead and did it anyway :)

Reply:
Bumping to v4 actually threw up a different error, which I think is related to the package* rather than anything we're doing, so I used the workaround referenced [here](https://github.com/broofa/mime/issues/300#issuecomment-1909792619).

* not sure on specifics, but basically they've tried to drop support for commonJS and go pure ESM, but I think they haven't set it up quite right (e.g. could be something as simple as `allowJs` missing from the [tsconfig](https://github.com/broofa/mime/blob/main/tsconfig.json)), resulting in ye olde 'Cannot use import statement outside a module' error